src/main/webapp/app/entities/programming-exercise/programming-exercise.component.html

```diff
@@ -50,21 +54,25 @@ <h4>
         <table class="table table-striped">
             <thead>
                 <tr jhiSort [(predicate)]="predicate" [(ascending)]="reverse" [callback]="callback">
-                    <th jhiSortBy="id"><span jhiTranslate="global.field.id">ID</span>&nbsp;<fa-icon [icon]="'sort'"></fa-icon></th>
+                    <th class="d-none d-md-table-cell" jhiSortBy="id"><span jhiTranslate="global.field.id">ID</span>&nbsp;<fa-icon [icon]="'sort'"></fa-icon></th>
```

Comment:
please document these changes in the pull request

Reply:
Mentioned these changes in the PR doc